AutomatedTesting/Gem/PythonTests/PythonAssetBuilder/AssetBuilder_test_case.py

```diff
@@ -10,21 +10,25 @@
 import azlmbr.math
 import azlmbr.legacy.general
 
-def raise_and_stop(msg):
-    print (msg)
-    azlmbr.editor.EditorToolsApplicationRequestBus(azlmbr.bus.Broadcast, 'ExitNoPrompt')
+print('Starting mock asset tests')
+azlmbr.legacy.general.idle_enable(True)
+azlmbr.legacy.general.idle_wait(1.0)
```

Comment:
What is the wait for? Is there some kind of notification we can listen for instead?

Reply:
the logs for the Editor.log is stopping right before starting up the RHI so I want the Editor to get cooking then do a the check to make sure all the main systems are ready

Reply:
Spoke offline, I think we can listen for the ```NotifyMainWindowInitialized``` from ```AzToolsFramework::EditorEvents```. If that doesn't work, then we can add a new event that can be broadcast at the very end of the Editor startup.

Reply:
Updated